frontend/packages/dev-console/src/components/topology/components/nodes/PodSet.tsx

```diff
@@ -51,6 +69,16 @@ const PodSet: React.FC<PodSetProps> = ({ size, data, x = 0, y = 0 }) => {
         outerRadius={podStatusOuterRadius}
         data={completedDeploymentData}
         size={size}
+        subTitle={showPodCount && subTitle}
+        {...showPodCount &&
+          !accessAllowed && {
+            subTitleComponent: <ChartLabel style={{ fontSize: '14px' }} />,
+          }}
+        title={showPodCount && title}
+        {...showPodCount &&
+          !obj.status.availableReplicas && {
+            titleComponent: <ChartLabel style={{ fontSize: '14px' }} />,
```

Comment:
Use this `var(--pf-chart-global--FontSize--sm)`

Reply:
fixed in https://github.com/openshift/console/pull/3746/commits/7f87818e648f95f715c8cf0c640cae70b23df53f